configure.ac

```diff
@@ -463,7 +462,9 @@ darwin*)
 esac
 
 dnl Eventually remove this
-CFLAGS="$CFLAGS -I${prefix}/include/heartbeat"
+if test "$cross_compiling" != "yes"; then
```

Comment:
It is defined by /usr/share/autoconf/autoconf/general.m4 on my machine so I think it is a standard variable. That being said, I consider this patch as a hack because Pacemaker should detect the heartbeat include directory (and heartbeat should provide a way for that; probably with pkg-config).